### PR TITLE
fixed #83

### LINKED ISSUE #83: Date issue
I was testing the query 'quinta que vem' in portuguese

It should pick up Thursday on the next week. Not the next Thursday.

### PR DESCRIPTION
**Pull request by guivn**

### CODE REVIEW

**Comment by blandinw** [comment]
Thanks for your contribution, I would just like some feedback from native speakers!

**Comment by guilha** [comment]
I can help, but I didn't understand what is the matter on this thread :)

**Comment by blandinw** [comment]
This pull request tries to fix #83.

This commit makes
```
"sexta que vem"
"sexta feira da semana que vem"
"sexta-feira da próxima semana"
```

equal to 2013/2/22 when the current date is Tuesday Feb 12, 2013

File: resources/duckling/corpus/pt.time.clj [hunk 1]
```diff
@@ -154,36 +154,41 @@
   "quarta da próxima semana"
   (datetime 2013 2 20 :day-of-week 3)
 
+  "sexta que vem" ; when today is Tuesday, "mercredi prochain" is tomorrow
+  "sexta feira da semana que vem"
+  "sexta-feira da próxima semana"
+  (datetime 2013 2 22 :day-of-week 3)
```
Comment (blandinw) [inline resources/duckling/corpus/pt.time.clj]: even though this is not checked, can you make `:day-of-week` == 5 ?

**Pull request closed without merging**